lib/scss/components/news-list-item/_list-item.scss

```diff
@@ -44,24 +44,85 @@
 }
 
 .su-news-list {
-  @include margin(0);
-  @include padding(0);
-  @include modular-spacing('padding', 4 null);
-
   list-style: none;
   overflow: auto;
-  border-bottom: 1px solid $su-color-cool-grey-25;
+  padding-left: 0;
+
+  li {
+    &:first-of-type {
+      > article {
```

Comment:
How does this selector work? there's no article within a list.

Reply:
Each of the news articles are within an `<li>`. I'm using padding-top for spacing so on the first one I want to remove that top padding. Each one is in a list so that the screen readers will say you are on a list and then goes into each article.

Reply:
we don't currently have any `li > article` elements. how is this getting applied?

Reply:
Here is where I mean: 
![image](https://user-images.githubusercontent.com/1187335/120714398-ffff3600-c477-11eb-943a-ffaa2685000d.png)

Reply:
I don't have that same markup or styles. is there a change to config necessary? like a view update or something?

Reply:
Yes there was a profile change. I see now that the link to the profile change is further down in Associated Issues. 
https://github.com/SU-SWS/stanford_profile/pull/425

Reply:
Ahh. Thank you!